ale/drivers/base.py

```diff
@@ -369,109 +368,6 @@ def sun_position(self):
         }
 
 
-=======
-
-    @property
-    def label(self):
-        if not hasattr(self, "_label"):
-            if isinstance(self._file, pvl.PVLModule):
-                self._label = self._file
-            try:
-                self._label = pvl.loads(self._file)
-            except Exception:
-                self._label = pvl.load(self._file)
-            except:
-                raise ValueError("{} is not a valid label".format(self._file))
-        return self._label
-
-    @property
-    def file(self):
-        return self._file
-
-    @property
-    def interpolation_method(self):
-        return "lagrange"
-
-    @property
-    def starting_detector_line(self):
-        return 1
-
-    @property
-    def starting_detector_sample(self):
-        return 1
-
-    @property
-    def detector_sample_summing(self):
-        return 1
-
-    @property
-    def detector_line_summing(self):
-        return 1
-
-    @property
-    def name_platform(self):
-        return "Generic Platform"
-
-    @property
-    def name_sensor(self):
-        return "Generic Sensor"
-
-    @property
-    def radii(self):
-        return {
-            "semimajor" : self._semimajor,
-            "semiminor" : self._semiminor,
-            "unit" : "m" # default to meters
-        }
-
-    @property
-    def reference_height(self):
-        # TODO: This should be a reasonable #
-        return {
-            "minheight" : 0,
-            "maxheight": 1000,
-            "unit": "m"
-        }
-
-    @property
-    def focal_length_model(self):
-        return {
-            "focal_length" : self._focal_length
-        }
-
-    @property
-    def detector_center(self):
-        if not hasattr(self, '_detector_center'):
-            self._detector_center = {
-                "line" : self._detector_center_line,
-                "sample" : self._detector_center_sample
-            }
-        return self._detector_center
-
-    @property
-    def sensor_position(self):
-        return {
-            "positions" : self._sensor_position,
-            "velocities" : self._sensor_velocity,
-            "unit" : "m"
-        }
-
-    @property
-    def sensor_orientation(self):
-        return {
-            "quaternions" : self._sensor_orientation
-        }
-
-    @property
-    def sun_position(self):
-        return {
-            "positions" : self._sun_position,
-            "velocities" : self._sun_velocity,
-            "unit" : "m"
-        }
-
-
```

Comment:
Is all of this supposed to be pulled?

Reply:
I forgot to push some files that had merge conflicts dealt with.